packages/bp/src/admin/ui/src/workspace/collaborators/index.tsx

```diff
@@ -35,7 +36,7 @@ const List: FC<Props> = props => {
 
   const reloadUsers = () => {
     const filteredRoles = props.roles.map(x => x.id).join(',')
-    props.fetchUsers(filteredRoles)
```

Comment:
this will always add chatusers, isn't there a way we add this in props.roles ? where is props.roles set ? is it the user that sets it or is it static ?

Reply:
@EFF Hmm, from what I see these `roles` are taken from the workspaces.json and by default `chatUsers` are not usually defined there since they are defined internally and could be added via bp [sdk](https://botpress.com/reference/interfaces/_botpress_sdk_.addworkspaceuseroptions.html) as a workspace user. So, here the `roles` don't contain those and I followed logic in `UserList/index.tsx` which adds up `CHAT_USER_ROLE` manually when filtering the users returned from the backend. Let me know if there is another way to do it, I can refactor that then!

Reply:
I think that this should be handled in the backend. Maybe [here](https://github.com/botpress/botpress/blob/master/packages/bp/src/admin/workspace/roles/roles-router.ts#L13)! This way, you wouldn't have to change anything in the front-end. What do you think?